client/src/app/site/pages/organization/pages/mediafiles/mediafiles.subscription.ts

```diff
@@ -1,27 +0,0 @@
-import { SubscriptionConfigGenerator } from 'src/app/domain/interfaces/subscription-config';
-
-import { ORGANIZATION_ID } from '../../services/organization.service';
-import { ViewOrganization } from '../../view-models/view-organization';
-
-export const ORGANIZATION_MEDIAFILE_LIST_SUBSCRIPTION = `organization_mediafile_list`;
-
-export const getOrganizationMediafileListSubscriptionConfig: SubscriptionConfigGenerator = () => ({
-    subscriptionName: ORGANIZATION_MEDIAFILE_LIST_SUBSCRIPTION,
-    modelRequest: {
-        viewModelCtor: ViewOrganization,
-        ids: [ORGANIZATION_ID],
-        follow: [
-            {
-                idField: `mediafile_ids`,
-                fieldset: `organizationDetail`,
-                follow: [
-                    {
-                        idField: `meeting_mediafile_ids`,
-                        fieldset: [`mediafile_id`],
-                        follow: [{ idField: `meeting_id`, fieldset: [`name`] }]
-                    }
-                ]
```

Comment:
The `getOrganizationSubscriptionConfig` is missing this follow. Therefore I think that this should not be deleted.